src/strategies/staked-defi-balance/index.ts

```diff
@@ -2,10 +2,14 @@
 
 import { formatUnits } from '@ethersproject/units';
 import { getAddress } from '@ethersproject/address';
-import { multicall } from '../../utils';
+import { multicall, getProvider } from '../../utils';
+import openStakingAbi from './ABI/openStakingABI.json';
+import standardStakingAbi from './ABI/standardStakingABI.json';
+import { ABI } from './types';
+
 
 export const author = 'taha-abbasi';
-export const version = '0.1.0';
+export const version = '1.2.6';
```

Comment:
```suggestion
export const version = '1.0.0';
```

Reply:
Needs to stay 1.2.6 to be synced with our internal semantic system